client/components/LearnerViewDirective.js

```diff
@@ -151,6 +147,14 @@ tie.directive('learnerView', [function() {
           $scope.$apply();
         };
 
+        $scope.codeMirrorOptions = {
+          autofocus: true,
+          lineNumbers: true,
+          mode: LANGUAGE_PYTHON,
```

Comment:
Might be worth adding a TODO to eventually support more languages, but this is a nit, _at best_.